applications/segment_everything/operators.py

```diff
@@ -0,0 +1,389 @@
+import datetime
+
+import cupy as cp
+import cupyx.scipy.ndimage
+import holoscan as hs
+import numpy as np
+from holoscan.core import Operator, OperatorSpec
+from holoscan.gxf import Entity
+from utils import CupyArrayPainter, DecoderInputData, PointMover, save_cupy_tensor
+
+
+class DecoderConfigurator(Operator):
```

Comment:
Could you please provide a brief description of each operator in a header comment? Would make it easier to understand on first read what each pipeline component is doing